suites/quincy/nfs/tier-2_nfs_ganesha_spec_storage.yaml

```diff
@@ -0,0 +1,159 @@
+              command: apply
+              service: rgw
+              pos_args:
+                - rgw.1
+              args:
+                placement:
+                  label: rgw
+          - config:
+              args:
+                - "ceph fs volume create cephfs"
+              command: shell
+          - config:
+              args:
+                placement:
+                  label: mds
+              base_cmd_args:
+                verbose: true
+              command: apply
+              pos_args:
+                - cephfs
+              service: mds
+          - config:
+              args:
+                - "ceph osd pool create rbd"
+              command: shell
+          - config:
+              args:
+                - "rbd pool init rbd"
+              command: shell
+      desc: bootstrap and deploy services.
+      destroy-cluster: false
+      polarion-id: CEPH-83573713
+      module: test_cephadm.py
+      name: Deploy cluster using cephadm
+
+  - test:
+      abort-on-fail: true
+      config:
+        command: add
+        id: client.1
+        node: node4
+        install_packages:
+          - ceph-common
+          - ceph-fuse
+        copy_admin_keyring: true
+      desc: Configure the RGW,RBD client system
+      destroy-cluster: false
+      module: test_client.py
+      name: configure client
+
+  - test:
+      abort-on-fail: true
+      config:
+        command: add
+        id: client.2
+        node: node5
+        install_packages:
+          - ceph-common
+          - ceph-fuse
+        copy_admin_keyring: true
+      desc: Configure the RGW,RBD client system
+      destroy-cluster: false
+      module: test_client.py
+      name: configure client
+
+  - test:
+      abort-on-fail: true
+      config:
+        command: add
+        id: client.3
+        node: node6
+        install_packages:
+          - ceph-common
+          - ceph-fuse
+        copy_admin_keyring: true
+      desc: Configure the RGW,RBD client system
+      destroy-cluster: false
+      module: test_client.py
+      name: configure client
+
+  - test:
+      abort-on-fail: true
+      config:
+        command: add
+        id: client.4
+        node: node7
+        install_packages:
+          - ceph-common
+          - ceph-fuse
+        copy_admin_keyring: true
+      desc: Configure the RGW,RBD client system
+      destroy-cluster: false
+      module: test_client.py
+      name: configure client
+
+  - test:
+      name: Test nfs server with SPECstorage workload SWBUILD
+      module: nfs_spec_storage_tool.py
+      desc: Test nfs server performance with SPECstorage workload SWBUILD
+      polarion-id:
+      abort-on-fail: false
+      config:
+        mount_point: /mnt/nfs
+        nfs_version: 4.1
+        benchmark: SWBUILD
+        load: 1
+        incr_load: 1
+        num_runs: 1
+        clients:
+          - client1
+          - client2
+          - client3
+          - client4
+        benchmark_defination:
```

Comment:
Change keyword to `benchmark`

Reply:
Hi @vamahaja 

We are already using "benchmark" keyword for name of benchmark
https://github.com/red-hat-storage/cephci/pull/2831/files#diff-5b5af8b197fcc455e6d57222bdbcebec866294024feb682be9362790f3dfe00cR145

"benchmark_defination" is correct since we by defult in spec storage tool we have same keyword so it is easily for user to understand.